### PR TITLE
Contributor Guide and support materials

### PR DESCRIPTION
**Pull request by thitch97**
[Readable](https://github.com/thitch97/rfcs/blob/master/accepted/0000-contributing-guidelines.md)

### CODE REVIEW

**Comment by sophiewigmore** [comment]
A suggestion I have would be to link out to where a user can view the list of maintainers/contributors for a language family, such as https://github.com/orgs/paketo-buildpacks/teams/nodejs/teams maybe? It could work in the "Where can I reach out to the team?" section or maybe in the contributor section.

File: accepted/0000-template.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# {{TITLE: a human-readable title for this RFC!}}
```
Comment (nebhale) [inline accepted/0000-template.md:1]: @thitch97 Why is this file added/changed?  I’d expect that, given the contents of `main`, that this file should be unchanged.

Comment (thitch97) [reply accepted/0000-template.md]: I removed the template file by mistake in my original commit and I was attempting to rectify that. I've fixed the issue now.

**Review by nebhale — APPROVED** [review]

**Review by sclevine — APPROVED** [review]
